src/modules/permissions/infrastructure/repositories/permission.server.repository.ts

```
@@ -102,6 +102,16 @@ export class PermissionServerRepository
     await super.delete({ serverId, roleId });
   }
 
+  async deleteById(id: string): Promise<void> {
+    const result = await this.delete({ id });
+    console.log(`Deleted server permission with id ${id}, result:`, result);
+  }
+
+  async deleteByRoleId(roleId: string): Promise<void> {
+    const result = await this.delete({ roleId });
+    console.log(`Deleted all server permissions for role ${roleId}, result:`, result);
```

ESLint

[error] Replace ``Deleted·all·server·permissions·for·role·${roleId},·result:`,·result` with `⏎······`Deleted·all·server·permissions·for·role·${roleId},·result:`,⏎······result,⏎····`

(prettier/prettier)